test/Microsoft.DocAsCode.Dotnet.Tests/SymbolUrlResolverUnitTest.cs

```diff
@@ -84,18 +85,45 @@ public static void GetPdbSourceLinkUrlTest()
         var type = assembly.GetTypeByMetadataName(typeof(DotnetApiCatalog).FullName);
         Assert.NotNull(type);
         Assert.Equal(
-            "https://github.com/dotnet/docfx/blob/*/src/Microsoft.DocAsCode.Dotnet/DotnetApiCatalog.cs",
+            @$"{GetGitRepoOrigin()}/blob/*/src/Microsoft.DocAsCode.Dotnet/DotnetApiCatalog.cs",
             ReplaceSHA(SymbolUrlResolver.GetPdbSourceLinkUrl(compilation, type)));
 
         var method = type.GetMembers(nameof(DotnetApiCatalog.GenerateManagedReferenceYamlFiles)).FirstOrDefault();
         Assert.NotNull(method);
         Assert.Equal(
-            "https://github.com/dotnet/docfx/blob/*/src/Microsoft.DocAsCode.Dotnet/DotnetApiCatalog.cs",
+            @$"{GetGitRepoOrigin()}/blob/*/src/Microsoft.DocAsCode.Dotnet/DotnetApiCatalog.cs",
```

Comment:
May be we can simply change the assertion to say the expected url starts with `https://github.com/` and ends with `/blob/*/...`?

Reply:
Agreed - that change is now in the latest commit.